data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -998,6 +998,9 @@ spec:
               openstack:
                 description: OpenStack is the configuration used when installing on OpenStack.
                 properties:
+                  apiFloatingIP:
+                    description: APIFloatingIP is the IP address of an available floating IP in your OpenStack cluster to associate with the OpenShift API.
```

Comment:
nit: `with the OpenShift API.` -> `with the OpenShift API load balancer.`?

Reply:
done